content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -97,7 +97,7 @@ <h3>Services and Labels</h3>
 				<h3>Create a new Service</h3>
 				<p>Let’s verify that our application is running. We’ll use the <code>kubectl get</code> command and look for existing Pods:</p>
 				<p><code><b>kubectl get pods</b></code></p>
-				<p>If no pods are running then it means the interactive environment is still reloading its previous state. Please wait a couple of seconds and list the Pods again. You can continue once you see the one Pod running.</p>
+				<p>If no Pods are running then it means the objects from the previous tutorials were cleaned up. In this case, go back and recreate the tutorial Deployment. Please wait a couple of seconds and list the Pods again. You can continue once you see the one Pod running.</p>
```

Comment:
Would it be beneficial to include a hyperlink in the text ‘tutorial Deployment’ that directs to the corresponding tutorial page?

Reply:
I think it would, yes. I'll update to include the link.

Reply:
Updated!